data/maps/TradeCenter/map.json

```diff
@@ -46,22 +46,24 @@
   ],
   "coord_events": [
     {
+      "comment": "This is trade seat #1.",
```

Comment:
Let's not add any custom "comment" fields at this time.  The name of the event script should be sufficient.

Reply:
My bad; that was for personal use while I was reverse-engineering. Removed.